lib/analysis/network.py

```diff
@@ -419,9 +423,9 @@ def give_userlist_where_degree(degree_dict, degree):
     nodes_with_IN_degree.insert(0, ["total_nodes", sum(data[1] for data in nodes_with_IN_degree)])
     nodes_with_TOTAL_degree.insert(0, ["total_nodes", sum(data[1] for data in nodes_with_TOTAL_degree)])
 
-    raw_out = []
-    raw_in = []
-    raw_total = []
+    raw_out = [str(date)]
+    raw_in = [str(date)]
```

Comment:
will the dates be correct if for example analysis was for 2 months from Jan to Feb?